### PR TITLE
Add youtube video and fix URLs

### LINKED ISSUE #6: Add youtube video to README
Should upload the project video (created during the course) to YouTube, and link to it in the README file.

### PR DESCRIPTION
**Pull request by haakjell**
* Add a link to the youtube video in the description part of the readme. (Resolves #6)
* Fix the paths to the animations such that they now link directly to files in repo rather than to imgur. (This was possible due to the repo now being public)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 This project was part of the Artificial Intelligence Programming (IT3105) course at NTNU spring 2022. The aim of this project was to create a reinforcement learning system using the SARSA RL Algorithm to learn to play a simple physics game. The AI has been trained to play a game, which we will refer to as the *The Acrobat Game*. The acrobat game is a simulated game designed to approximate an acrobat holding on to a horizontal bar using their hands.
 
+[This youtube video](https://youtu.be/fcoKjOAXDZ0) briefly explainins the project and its results.
```
Comment (AnmolS99) [inline README.md:7]: *explains

File: README.md [hunk 2]
```diff
@@ -40,8 +42,8 @@ pip install tensorflow numpy matplotlib
 
 The AI's performance before any training:
 
-![Animation showing perfomance before training](https://i.imgur.com/9vloaDx.gif)
+![Animation showing perfomance before training](./images/1.gif)
```
Comment (AnmolS99) [inline README.md:45]: GIF does not show in preview mode.

File: README.md [hunk 3]
```diff
@@ -40,8 +42,8 @@ pip install tensorflow numpy matplotlib
...
 
 After training:
 
-![Animation showing perfomance after training](https://i.imgur.com/mjYm9I9.gif)
+![Animation showing perfomance after training](./images/2.gif)
```
Comment (AnmolS99) [inline README.md:49]: GIF does not show in preview mode.

**Review by AnmolS99 — CHANGES_REQUESTED** [review]
GIF don't show for some reason.

File: README.md (hunk 1 above)
Comment (haakon8855) [reply README.md:7]: haha

File: README.md (hunk 2 above)
Comment (haakon8855) [reply README.md:45]: jo

Comment (AnmolS99) [reply README.md:45]: You are right. It works for me too.

**Review by AnmolS99 — APPROVED** [review]
LGTM!

**Pull request merged**